sources/Renderer/OpenGL/Shader/GLShaderProgram.cpp

```diff
@@ -482,7 +482,7 @@ static void GLQueryVertexAttributes(GLuint program, ShaderReflection& reflection
                 return true;
             if (lhs.location > rhs.location)
                 return false;
-            return (lhs.name < rhs.name);
+            return lhs.name.compare(rhs.name) < 0;
```

Comment:
This change made me add new operators in `<String.h>` with bdcd2ba, so this line won't have to be changed anymore.